mobile-app/app/screens/AppNavigator/screens/Portfolio/hooks/EvmTokenBalances.ts

```diff
@@ -106,7 +108,7 @@ export function useEvmTokenBalances(): { evmTokens: WalletToken[] } {
     if (isFocused && domain === DomainType.EVM) {
       getEvmTokens();
     }
-  }, [evmAddress, blockCount, isFocused, domain]);
+  }, [evmAddress, blockCount, isFocused, domain, evmWalletDetails]);
```

Comment:
i guess we dont need `evmAddress, blockCount, isFocused, domain` here, instead lets add this in above batch dispatch

Reply:
hmm but above batch is setting the `evmWalletDetails` data, which should be a dependency here 🤔

Reply:
oh ok, sorry misread the comment. let me update